samples/test/support/action/clickLoginWithFacebookInWidget.ts

```diff
@@ -0,0 +1,7 @@
+import clickElement from './clickElement';
+import { OktaSignInV1, OktaSignInOIE } from  '../selectors';
+
+export default async () => {
+  const OktaSignIn = process.env.ORG_OIE_ENABLED ? OktaSignInOIE : OktaSignInV1;
```

Comment:
Looks like this logic appears in lots of places, how about extract to a util function something like `getOktaSignInForm`?

Reply:
Done